dna/zomes/projects/src/project/goal/validate.rs

```diff
@@ -11,6 +11,8 @@ use hdk::prelude::*;
 use hdk_crud::WrappedAgentPubKey;
 
 #[hdk_extern]
+/// Creates only allowed if `user_hash` of the Goal entry matches the agent
+/// creating the entry, unless the Goal is imported 
```

Comment:
Can you also add that: `user_edit_hash` must not be set (before "unless the Goal is imported")

Reply:
```suggestion
/// creating the entry and if `user_edit_hash` is not set, unless the Goal is imported 
```